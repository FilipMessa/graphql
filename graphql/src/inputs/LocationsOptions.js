// @flow

import { GraphQLInputObjectType } from 'graphql';
import GraphQLLocale from '../inputs/Locale';

export default new GraphQLInputObjectType({
  name: 'LocationsOptionsInput',
  fields: {
    locale: {
      type: GraphQLLocale,
    },
  },
});
