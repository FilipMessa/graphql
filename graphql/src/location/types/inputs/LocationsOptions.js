// @flow

import { GraphQLInputObjectType } from 'graphql';
import GraphQLLocale from '../../../common/types/enums/Locale';

export default new GraphQLInputObjectType({
  name: 'LocationsOptionsInput',
  fields: {
    locale: {
      type: GraphQLLocale,
    },
  },
});
