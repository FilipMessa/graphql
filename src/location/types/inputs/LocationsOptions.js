// @flow

import { GraphQLInputObjectType } from 'graphql';
import GraphQLLocale from '../../../common/types/enums/Locale';
import GraphQLLocationType from './LocationType';

export default new GraphQLInputObjectType({
  name: 'LocationsOptionsInput',
  fields: {
    locale: {
      type: GraphQLLocale,
      description: 'Language tag in the format of the RFC 5646.',
    },
    locationType: {
      type: GraphQLLocationType,
      description: 'Desired location type the response should only include.',
    },
  },
});
