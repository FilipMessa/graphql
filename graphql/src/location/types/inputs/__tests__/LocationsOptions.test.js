// @flow

import LocationsOptions from '../LocationsOptions';

it('LocationsOptions type should have valid fields', () => {
  expect(LocationsOptions.getFields()).toMatchSnapshot();
});
