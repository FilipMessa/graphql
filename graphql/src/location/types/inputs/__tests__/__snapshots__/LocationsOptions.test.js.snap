// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocationsOptions type should have valid fields 1`] = `
Object {
  "locale": Object {
    "description": "Language tag in the format of the RFC 5646.",
    "name": "locale",
    "type": "Locale",
  },
  "locationType": Object {
    "description": "Desired location type the response should only include.",
    "name": "locationType",
    "type": "LocationType",
  },
}
`;
